### PR TITLE
27-28-w01-d04-hw  - events Lab

### PR DESCRIPTION
**Pull request by tgaff**
[Completes #18]

### CODE REVIEW

File: homework/27-28/week-01/README.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ Please use any remaining time to complete and review the exercises from dawn & d
...
+1. **Reading**: *15 minutes* When a web browser displays a page, it's not just showing the contents of an HTML file. It also creates a structure that JavaScript can interact with. This structure is called the Document Object Model (DOM). Unlike the HTML file, the DOM can change over time -- and we can change it with JavaScript! The DOM is vitally important for dynamic web pages where elements are added, removed, or changed over time. **Read this [brief introduction to the DOM](http://docs.webplatform.org/wiki/dom/tutorials/introduction).**
 
-1. **Activity**: *45 minutes* Open the Chrome dev tools with `Cmd` + `Opt` + `i`, and find the Elements tab.
```
Comment (tgaff) [inline homework/27-28/week-01/README.md:34]: white-space auto-destroyed!

File: homework/27-28/week-01/README.md [hunk 2]
```diff
@@ -48,10 +48,8 @@ Please use any remaining time to complete and review the exercises from today:
 <!--
 ## Day 4
 
-1. Reading
-2. Friday Review Prep
-    - Complete the [Week 1 Self-Assessment](#PENDING) and identify 2 topics you want to review tomorrow
```
Comment (nathanallen) [inline homework/27-28/week-01/README.md]: I'd like to keep these two items

File: homework/27-28/week-01/README.md [hunk 3]
```diff
@@ -48,10 +48,8 @@ Please use any remaining time to complete and review the exercises from today:
...
-    - Ask and/or upvote 3 questions on QuestionCookie: http://www.questioncookie.com/wdi-27-28-w1-review
```
Comment (nathanallen) [inline homework/27-28/week-01/README.md:54]: keep

Comment (tgaff) [inline homework/27-28/week-01/README.md:54]: Where is the week 1 self-assessment?
@nathanallen

Comment (nathanallen) [inline homework/27-28/week-01/README.md:54]: I need to migrate the assessment questions from the spreadsheet to a form. This is rather laborious as it has to be done by hand, for each class.

Comment (nathanallen) [inline homework/27-28/week-01/README.md:54]: @tgaff feel free to merge, I can add it later.

**Pull request merged**